internal/cmd/module/flags.go

```diff
@@ -8,6 +8,12 @@ var flagModuleID = &cli.StringFlag{
 	Required: true,
 }
 
+var flagVersionID = &cli.StringFlag{
+	Name:     "versionid",
```

Comment:
not a fan of this, but life is life. unless someone has a better idea?

Reply:
Maybe we could provide the version itself, not the ID?

Reply:
that would require changes in the backend, so i'd rather not do that :/